test/functional/test_framework/util.py

```diff
@@ -201,6 +201,14 @@ def str_to_b64str(string):
 def satoshi_round(amount):
     return Decimal(amount).quantize(Decimal('0.00000001'), rounding=ROUND_DOWN)
 
+def no_exception(predicate, action):
+    try:
+        predicate()
+    except Exception as e:
```

Comment:
I'd prefer to pass in the exception here over a wildcard . Or, since this is only used once, could just make the function `def` inline in test_framework.py